code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -276,7 +276,7 @@
 
 		add_fingerprint(usr)
 
-		if(!prevent_warning && !istype(W, /obj/item/weapon/gun/projectile/automatic/crossbow))
+		if(!prevent_warning && !istype(W, /obj/item/weapon/gun/energy/kinetic_accelerator/crossbow))

```

Comment:
I didnt invent this feature for storage it was just a path replacement

But I agree and might change it when fixing the conflicts after steelpoint's ERT PR is merged (someone do that)